services/app-web/src/pages/StateSubmission/Documents/Documents.test.tsx

```diff
@@ -944,45 +944,6 @@ describe('Documents', () => {
     })
 
     describe('Document categories checkbox', () => {
-        it('not present on contract only submission, categories default to contract-supporting', async () => {
```

Comment:
Did you mean to comment this out? Seems like a valid test

Reply:
it's back